platform/ext/target/nordic_nrf/common/core/CMakeLists.txt

```diff
@@ -162,6 +171,7 @@ if(BL2)
         PRIVATE
             cmsis_drivers/Driver_Flash.c
             cmsis_drivers/Driver_USART.c
+            $<$<C_COMPILER_ID:GNU>:${CMAKE_CURRENT_SOURCE_DIR}/startup.c>
```

Comment:
This file should be added to `bl` library (same as the board specific startup files).

Reply:
fixed